apps/drec-api/migrations/1636639831399-DeviceGroupBuyerId.ts

```diff
@@ -5,19 +5,13 @@ export class DeviceGroupBuyerId1636639831399 implements MigrationInterface {
 
   public async up(queryRunner: QueryRunner): Promise<void> {
     await queryRunner.query(
-      `ALTER TABLE "device_group" DROP COLUMN "buyerAddress"`,
-    );
-    await queryRunner.query(
-      `ALTER TABLE "device_group" ADD "buyerAddress" text`,
+      `ALTER TABLE "device_group" ALTER COLUMN "buyerAddress" TYPE text`,
     );
   }
 
   public async down(queryRunner: QueryRunner): Promise<void> {
     await queryRunner.query(
-      `ALTER TABLE "device_group" DROP COLUMN "buyerAddress"`,
-    );
-    await queryRunner.query(
-      `ALTER TABLE "device_group" ADD "buyerAddress" character varying`,
+      `ALTER TABLE "device_group" ALTER COLUMN "buyerAddress" character varying`,
```

Comment:
Are you sure this works? The syntax looks incorrect.

Reply:
I tested it and didn`t got an error. But you`re right, it looks weird. I reverted the down function

Reply:
So you tested a downgrade? With which data? From some prod system.

The solution is not too revert, but to fix the syntax. You missed the keyword **TYPE**.

I guess this is the correct syntax
```sql
ALTER TABLE "device_group" ALTER COLUMN "buyerAddress" TYPE character varying
```